cmd/kubeadm/app/constants/constants.go

```diff
@@ -143,7 +143,7 @@ const (
 	// the TLS bootstrap to get itself an unique credential
```

Comment:
vanduc95 there are a lot more Master entries in this file.
we need to change code as well not only comments!